webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/inventory/InventoryFileWatcher.scala

```diff
@@ -79,10 +79,15 @@ final class Watchers(incoming: FileMonitor, updates: FileMonitor) {
 }
 object Watchers {
 
+  val inventoryExtentions = "gz" :: "xml" :: "ocs" :: "sign" :: Nil
```

Comment:
it should be a set, it's done for that - but actually, for 4 elements, it's still special cased so it shouldn't matter at all.